grpc.go

```diff
@@ -892,10 +892,6 @@ func (a *agentGRPC) DestroySandbox(ctx context.Context, req *pb.DestroySandboxRe
 	}
 	a.sandbox.Unlock()
 
-	if err := a.sandbox.removeNetwork(); err != nil {
```

Comment:
Based on the same logic, can we simplify `DestroySandbox` by removing `removeMounts` and `teardownSharedPidNs` as well? Assuming that we do not reuse a sandbox after destroying it, the only thing must being done here is necessary pre-condition check and syscall.Sync().

Reply:
We need to discuss about this, but if it's done, it's gonna be done in a follow up PR. This PR is more about the network.

Reply:
Yes, agreed.